### PR TITLE
docs: add missing docs for array rule

### PR DESCRIPTION
**Pull request by zj9495**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] Bug fix
- [ ] New feature
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
Issue https://github.com/ant-design/ant-design/issues/27287  
Code https://github.com/react-component/field-form/blob/master/src/utils/validateUtil.ts#L78

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/form/index.en-US.md](https://github.com/zj9495/ant-design/blob/docs-form/components/form/index.en-US.md)
[View rendered components/form/index.zh-CN.md](https://github.com/zj9495/ant-design/blob/docs-form/components/form/index.zh-CN.md)

### CODE REVIEW

**Comment by zj9495** [comment]
Hi @afc163, could u pls review this PR? Thanks!

File: components/form/index.zh-CN.md [hunk 1]
```diff
@@ -318,6 +318,7 @@ type Rule = RuleConfig | ((form: FormInstance) => RuleConfig);
 | validateTrigger | 设置触发验证时机，必须是 Form.Item 的 `validateTrigger` 的子集 | string \| string\[] |
 | validator | 自定义校验，接收 Promise 作为返回值。[示例](#components-form-demo-register)参考 | ([rule](#Rule), value) => Promise |
 | whitespace | 如果字段仅包含空格则校验不通过 | boolean |
+| defaultField | 仅在`type`为`array`类型时有效，用于校验 array 元素的规则 | [rule](#Rule) |
```
Comment (afc163) [inline components/form/index.zh-CN.md:321]:
```suggestion
| defaultField | 仅在 `type` 为 `array` 类型时有效，用于校验 array 元素的规则 | [rule](#Rule) |
```

File: components/form/index.en-US.md [hunk 2]
```diff
@@ -316,6 +316,7 @@ type Rule = RuleConfig | ((form: FormInstance) => RuleConfig);
 | validateTrigger | Set validate trigger event. Must be the sub set of `validateTrigger` in Form.Item | string \| string\[] |
 | validator | Customize validation rule. Accept Promise as return. See [example](#components-form-demo-register) | ([rule](#Rule), value) => Promise |
 | whitespace | Failed if only has whitespace | boolean |
+| defaultField | Only valid when `type` is `array`, used to validate the rules of array elements | [rule](#Rule) |
```
Comment (afc163) [inline components/form/index.en-US.md:319]:
```suggestion
| defaultField | Validate rule for all array elements, valid when `type` is `array` | [rule](#Rule) |
```

Comment (afc163) [reply components/form/index.en-US.md:319]: 还有个 fields 可以写一下：https://github.com/yiminghe/async-validator/#deep-rules

**Comment by afc163** [comment]
ping @zj9495

**Pull request merged**